src/functions/chatPostMessage.js

```diff
@@ -0,0 +1,34 @@
+const dotenv = require("dotenv");
+const https = require("https");
```

Comment:
can you use import?

Reply:
I modified it and created another pull request #39 
Could you check the code modification in the new pull request?